examples/gitops-kubevirt/04-upgrades/monitoring/check-upgrade-status.sh

```
@@ -0,0 +1,307 @@
+    local last_update=$(get_field hostedcluster "$NAMESPACE" "$CLUSTER_NAME" ".status.lastVersion.lastTransitionTime")
+    if [ "$last_update" != "N/A" ]; then
+        log_info "Last updated: $last_update"
+    fi
+
+    echo ""
+}
+
+check_nodepool() {
+    log_header "NodePool Status"
+
+    local nodepools=$(oc get nodepool -n "$NAMESPACE" -o jsonpath="{.items[*].metadata.name}" 2>/dev/null)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)